ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -924,6 +924,15 @@ tests:
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-powervs-ipi
   timeout: 6h0m0s
+- as: fips-payload-scan
+  steps:
+    cluster_profile: aws
+    env:
+      MAJOR_MINOR: "4.16"
+    test:
+    - ref: openshift-release-fips
+    - ref: clusterbot-wait
+    workflow: ipi-aws
```

Comment:
I would go ahead and add this configuration to all the versions where the job will be ran from.
After this PR merges, you can chase it with the necessary update to add informing jobs to the respective releases.